src/main/kotlin/com/github/biomejs/intellijbiome/GeneralProcessCommandBuilder.kt

```diff
@@ -48,9 +50,34 @@ class GeneralProcessCommandBuilder : ProcessCommandBuilder {
         command.withExePath(exec)
         workingDir?.let { command.withWorkingDirectory(Path(it)) }
         inputFile?.let { command.withInput(File(it.path)) }
-        command.addParameters(parameters)
         charset?.let { command.withCharset(it) }
 
-        return BiomeTargetRun.General(command)
+        // patch the command line to be run on WSL 2 instead of the host
+        val wslDistribution = WslPath.getDistributionByWindowsUncPath(exec)
+        if (wslDistribution != null) {
+            command.withExePath(wslDistribution.getWslPathSafe(Path(exec)))
```

Comment:
Now, we need to convert the executable path to UNIX path inside the WSL distribution, such as `/path/to/biome`.